test/ex_doc/autolink_test.exs

```diff
@@ -449,11 +450,17 @@ defmodule ExDoc.AutolinkTest do
     assert warn(~m"[bad](`String.upcase/9`)", extras: []) =~
              "documentation references \"String.upcase/9\" but it is undefined or private"
 
-    assert warn(~m"[custom text](`Elixir.Unknown`)", []) =~
-             "documentation references module \"Elixir.Unknown\" but it is undefined\n"
+    assert_and_warn(
```

Comment:
could you please revert this change? with this helper we're saving typing a few characters but it is less flexible, for example we can't easily inspect the whole output (which was as simple as `|> IO.inspect` before but now the helper returns a boolean).

fwiw I am the first to admit that I probably went a bit overboard with adding many test helpers (which are saving typing characters but add complexity) so let's not add even more.